src/client/components/cube-view/cube-view.tsx

```diff
@@ -301,11 +306,47 @@ export class CubeView extends React.Component<CubeViewProps, CubeViewState> {
     this.setState({ essence: newEsssence });
   }
 
+  getStoredLayout(): any {
+    let defaultLayout: any = {left: 240, right: 240};
+    let storedLayout: any = undefined;
+    if (localStorage.supported()) {
+      storedLayout = localStorage.get('cube-layout');
```

Comment:
could you change `cube-layout` to `cube-view-layout`, you'll see why soon. Muhahahaha